mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/checker/SkipSTInstallationChecker.java

```diff
@@ -0,0 +1,39 @@
+package org.arquillian.smart.testing.mvn.ext.checker;
+
+import org.apache.maven.project.MavenProject;
+import org.arquillian.smart.testing.configuration.Configuration;
+import org.arquillian.smart.testing.mvn.ext.dependencies.ExtensionVersion;
+
+import static org.arquillian.smart.testing.configuration.Configuration.SMART_TESTING_DISABLE;
+
+public class SkipSTInstallationChecker {
```

Comment:
This class name is very similar to: `SkipInstallationChecker` it's very confusing.
Please, merge these two classes. The constructor can stay same, you just add another method `shouldSkipForConfig(Configuration config)` which you will call at the places where you call your `SkipSTInstallationChecker`